core/storage/statistics/gae_models.py

```diff
@@ -717,22 +718,30 @@ class ExplorationStatsModel(base_models.BaseModel):
     exp_id = ndb.StringProperty(indexed=True)
     # Version of exploration.
     exp_version = ndb.IntegerProperty(indexed=True)
-    # Number of learners starting the exploration.
-    num_starts = ndb.IntegerProperty(indexed=True)
+    # Number of learners starting the exploration (v1 - historical data).
```

Comment:
Everything is "historical data". I think you need a better description of v1. Saying "for data collected before Nov 2017" is fine; you can elaborate more in feconf.

In general, make sure you have sufficient docstrings/detail so that a future dev, or future-you in a year's time, can look at this and easily grok what's going on.

Reply:
Done. Modified doc strings and added one in feconf as well.